tensorflow/inference/docker/2.10/py3/sdk2.8.0/Dockerfile.neuron

```diff
@@ -58,12 +58,12 @@ RUN apt-get update \
  && echo 'deb http://nginx.org/packages/ubuntu/ focal nginx' >> /etc/apt/sources.list \
  && apt-get update \
  && apt-get -y install --no-install-recommends \
-    nginx=1.20.1* \
-    nginx-module-njs=1.20.1* \
+    nginx=1.22.1* \
+    nginx-module-njs=1.22.1* \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
 
-# Install python3.9
+# Install Python3
```

Comment:
Check : why is this change needed?

Reply:
We're installing Python 3.8 with this image. I just wanted to keep it more generic to minimize future updates, I'll change it to the actual version number